apps/portal/app/account/_components/account.tsx

```diff
@@ -133,12 +133,12 @@ export function Account() {
       });
       return;
     }
-    setAccountSponsor(address, sponsorAddress);
+    setAccountSponsor(sponsorAddress);
```

Comment:
no more `from` param needed in anything...you'll see this refactored logic thoughout